### PR TITLE
map person to dynamics entity

### PR DESCRIPTION
**Pull request by rantifamilusi**
# Description

Map Person From contract to dynamics
[FAMS3-1400] (https://justice.gov.bc.ca/jira/browse/FAMS3-1400)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested? 

Unit test and test against dynamics

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## For bcgov contributors:

this PR fixes jira ticket: **FAMS3-1400**

### CODE REVIEW

File: app/DynamicsAdapter/DynamicsAdapter.Web.Test/Mapping/MappingProfileTest.cs [hunk 1]
```diff
@@ -334,6 +335,101 @@ public void PersonalPhoneNumber_should_map_to_SSG_PhoneNumber_correctly()
...
+            SSG_Person ssg_person = _mapper.Map<SSG_Person>(person);
+            Assert.AreEqual("FirstName", ssg_person.FirstName);
+            Assert.AreEqual("LastName", ssg_person.LastName);
+            Assert.AreEqual("MiddleName", ssg_person.MiddleName);
+            Assert.AreEqual("OtherName", ssg_person.ThirdGivenName);
+            Assert.AreEqual(null, ssg_person.DateOfDeathConfirmed);
+            Assert.AreEqual(new DateTime(2011, 1, 1), ssg_person.DateOfBirth);
+            Assert.AreEqual(new DateTime(2011, 1, 1), ssg_person.DateOfDeath);
+            Assert.AreEqual("M", ssg_person.Gender);
+            Assert.AreEqual(null, ssg_person.Incacerated);
```
Comment (peggy-gov-bc-ca) [inline app/DynamicsAdapter/DynamicsAdapter.Web.Test/Mapping/MappingProfileTest.cs]: nice

File: app/DynamicsAdapter/DynamicsAdapter.sln [hunk 2]
```diff
@@ -11,9 +11,14 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "test", "test", "{1BE19189-0
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DynamicsAdapter.Web.Test", "DynamicsAdapter.Web.Test\DynamicsAdapter.Web.Test.csproj", "{6283EFDD-21EC-4FE5-B01E-54588BBB8FCD}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Fams3Adapter.Dynamics", "Fams3Adapter.Dynamics\Fams3Adapter.Dynamics.csproj", "{64CD5D00-7677-4F3B-BBAA-8F70B68DB4A7}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Fams3Adapter.Dynamics", "Fams3Adapter.Dynamics\Fams3Adapter.Dynamics.csproj", "{64CD5D00-7677-4F3B-BBAA-8F70B68DB4A7}"
```
Comment (peggy-gov-bc-ca) [inline app/DynamicsAdapter/DynamicsAdapter.sln]: what's that?

File: app/DynamicsAdapter/DynamicsAdapter.sln [hunk 3]
```diff
@@ -11,9 +11,14 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "test", "test", "{1BE19189-0
...
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Fams3Adapter.Dynamics.Test", "Fams3Adapter.Dynamics.Test\Fams3Adapter.Dynamics.Test.csproj", "{F0A1BDFD-08C2-4A1E-A6CB-BD996B2274CF}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Fams3Adapter.Dynamics.Test", "Fams3Adapter.Dynamics.Test\Fams3Adapter.Dynamics.Test.csproj", "{F0A1BDFD-08C2-4A1E-A6CB-BD996B2274CF}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{44D4250C-B14B-49B8-8C85-A2373B2CF5FE}"
+	ProjectSection(SolutionItems) = preProject
+		.editorconfig = .editorconfig
```
Comment (peggy-gov-bc-ca) [inline app/DynamicsAdapter/DynamicsAdapter.sln]: and what's that?

File: app/DynamicsAdapter/DynamicsAdapter.sln (hunk 2 above)
Comment (rantifamilusi) [reply app/DynamicsAdapter/DynamicsAdapter.sln]: > looks good. Just not sure if sln, proj file can be checked in.

When project file have some changes they will be checked in. For example when we increase version number on contract. 

Is there a build issue on your end when you pull the branch and run it?

File: app/DynamicsAdapter/DynamicsAdapter.sln (hunk 3 above)
Comment (rantifamilusi) [reply app/DynamicsAdapter/DynamicsAdapter.sln]: it's a link to the .editorconfig in the solution. we should probably remove it

File: app/DynamicsAdapter/DynamicsAdapter.sln (hunk 2 above)
Comment (peggy-gov-bc-ca) [reply app/DynamicsAdapter/DynamicsAdapter.sln]: checked. it can build.

**Pull request merged**